### PR TITLE
python3Packages.rapidfuzz: 1.9.1 -> 2.0.2

### PR DESCRIPTION
**Pull request by dotlambda**
###### Motivation for this change
https://github.com/maxbachmann/RapidFuzz/releases/tag/v2.0.0

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by dotlambda** [comment]
Does someone know what I'm doing wrong?
```
CMake Error at /nix/store/irw4fr32wv87sq0d81ljxz9al30b7shq-python3.9-scikit-build-0.12.0/lib/python3.9/site-packages/skbuild/resources/cmake/targetLinkLibrariesWithDynamicLookup.cmake:579 (target_link_libraries):
  The keyword signature for target_link_libraries has already been used with
  the target "cpp_utils".  All uses of target_link_libraries with a target
  must be either all-keyword or all-plain.

  The uses of the keyword signature are here:

   * src/cython/CMakeLists.txt:8 (target_link_libraries)

Call Stack (most recent call first):
  /nix/store/irw4fr32wv87sq0d81ljxz9al30b7shq-python3.9-scikit-build-0.12.0/lib/python3.9/site-packages/skbuild/resources/cmake/FindPythonExtensions.cmake:427 (target_link_libraries_with_dynamic_lookup)
  src/cython/CMakeLists.txt:9 (python_extension_module)
```

File: pkgs/development/python-modules/rapidfuzz/default.nix [hunk 1]
```diff
@@ -2,34 +2,48 @@
...
   };
 
+  nativeBuildInputs = [
+    cmake
+    cython
+    scikit-build
+  ];
+
+  cmakeFlags = [
+    "-DCMAKE_MODULE_PATH=${scikit-build}/${python.sitePackages}/skbuild/resources/cmake"
```
Comment (dotlambda) [inline pkgs/development/python-modules/rapidfuzz/default.nix:36]: Should scikit-build install these files to `$out/share/cmake`?

**Comment by yuuyins** [comment]
@ofborg build test python3Packages.rapidfuzz

**Comment by dotlambda** [comment]
The error seems to be gone with he version of scikit-build on python-updates (https://github.com/NixOS/nixpkgs/pull/166489). However, scikit-build's test are broken on that branch. Help with fixing them is appreciated.

**Comment by dotlambda** [comment]
superseded by https://github.com/NixOS/nixpkgs/pull/168580

**Pull request closed without merging**